content/theme.ts

```diff
@@ -41,6 +121,9 @@ export const stakingTheme: Partial<ChakraTheme> = {
   components: {
     ...chakraTheme.components,
     Progress,
+    Button,
```

Comment:
If we want the buttons from the v3 theme I think we need to 
```
{ 
  Button: {
    ...chakraTheme.components.Button,
    ...Button
  }
```
Since object spread just merges one level deep